src/NodaTime.Demo/PeriodDemo.cs

```diff
@@ -86,6 +86,146 @@ public void ConstructionFromNanoseconds()
             Period period = Snippet.For(Period.FromNanoseconds(42));
             Assert.AreEqual(42, period.Nanoseconds);
             Assert.AreEqual("PT42n", period.ToString());
-        }        
+        }
+
+        [Test]
+        public void Between_LocalDatesDaysApart()
+        {
+            Period period = Snippet.For(Period.Between(
+                new LocalDate(2017, 11, 10),
+                new LocalDate(2017, 11, 15)));
+
+            Assert.Multiple(() =>
```

Comment:
I'd steer clear of Assert.Multiple in snippets. The snippet renderer isn't smart enough to understand that yet!
Just two separate assertions should be fine.

Reply:
Got it.